src/openeo_udf/api/datacube.py

```diff
@@ -5,6 +5,8 @@
 import numpy
 import xarray
 from typing import Dict, List
+import json
+from matplotlib import pyplot
```

Comment:
can you move this import inside the function? Importing matplotlib triggers quite a bit of things, and I've seen it fail in particular environments.

Reply:
Of course